cves/CVE-2019-19118.yml

```diff
@@ -49,7 +49,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Through the Django admin protal, after editing a child model, users were able 
+  to trigger a parent model's save() method even though the user only has view 
```

Comment:
Might be nice to add more information how users triggered the save() method by editing the child model for people not familiar with Django's admin portal.

Reply:
I see, more info would be helpful.
I've expanded a small amount on how the method was being triggered. Could you check again?